recipes-domx/meta-xt-prod-vgpu/recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -0,0 +1,5 @@
+# We need this header file to build gles-user-module. Since it was removed
+# by mesa recipe we have to install it manually.
+do_install_append () {
```

Comment:
you can also put a not, that for example, for AGL it is installed as a part of Qt in the same way

Reply:
Done